src/js/media-chrome-range.js

```diff
@@ -101,6 +103,52 @@ template.innerHTML = `
       ${trackStyles}
     }
 
+    #background,
+    #pointer {
+      ${trackStyles}
+      width: auto;
+      position: absolute;
+      top: 50%;
+      transform: translate(var(--media-range-track-translate-x, 0px), calc(var(--media-range-track-translate-y, 0px) - 50%));
```

Comment:
thought (non-blocking): in a [recent talk by Lea Verou](https://www.youtube.com/watch?v=ZuZizqDF4q8), she mentioned it might be better to keep variables unit agnostic as it'll make it more flexible. Then, you would apply the unit when you use the variable. Might be something to consider soonish as could be a breaking change.